src/cable/sagas.ts

```diff
@@ -1,41 +1,22 @@
 import { take, call, put } from 'redux-saga/effects'
 import { eventChannel } from 'redux-saga'
-import { songDeserializer, actions } from '../models/song'
+import { getSingleton } from './client'
 
 function initWebsocket() {
-
+  const client = getSingleton()
+  const ws = new WebSocket('ws://localhost:3000/cable')
```

Comment:
Should `'ws://localhost:3000/cable'` be stored as a CONSTANT somewhere, instead of being hardcoded? For example, what happens when we need to deploy the API?

Reply:
Yeah, eventually we'll have a "we need to operationalize this dang ol' musicbox" kind of deal where we pull these in to environment variables.